### PR TITLE
Monkeys can now throw shit at you

### PR DESCRIPTION
**Pull request by ghost**
ook ook

whuy

:cl: Thefastfoodguy
add: Monkeys can now throw shit at you
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
can we keep the residue meme

**Comment by PopNotes** [comment]
I feel it adds a layer of sophistication to the feature

**Comment by DinduStuffin** [comment]
it...

it's actually poo

+1

**Comment by KayeArray** [comment]
PEE PEE
POO POO

**Comment by doobiedoo23** [comment]
big huge -1

**Review by GuyonBroadway — APPROVED** [review]
Code looks ok... but it's POO.

**Comment by GuyonBroadway** [comment]
You should totally make it so poo patches are slippery for a while.

**Comment by DinduStuffin** [comment]
guys i got it

let's make poo work with the disgust system

File: hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//Poo (sorry -MRTY)
+/obj/effect/decal/cleanable/poo
+	name = "poo"
+	desc = "A pile of poo. Gross!"
+	icon = 'hippiestation/icons/effects/poo.dmi'
+	icon_state = "floor1"
+	random_icon_states = list("floor1", "floor2", "floor3", "floor4", "floor5", "floor6", "floor7")
+
+/obj/effect/decal/cleanable/poo/attack_hand(mob/user)
```
Comment (TDcoolguy300) [inline hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm:9]: Questionable

File: hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	user.visible_message("<span class='danger'>[user] puts their hand in the poo! Gross!</span>", "<span class='danger'>You put your hand in the poo, and immediatly regret it</span>")
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/mutable_appearance/poohands = mutable_appearance('hippiestation/icons/effects/poo.dmi', "poohands")
+		H.add_overlay(poohands)
+
+/obj/effect/decal/cleanable/poo/Initialize()
```
Comment (TDcoolguy300) [inline hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm:16]: No

**Comment by tortellinitony** [comment]
it's funny but I give a :-1:

**Comment by Pyko1** [comment]
much needed realism

File: hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/game/objects/effects/decals/Cleanable/poo.dm:16]: POOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by KayeArray** [comment]
It's alright when the monkeys aren't spammed everywhere

**Comment by PopNotes** [comment]
add this

**Comment by GuyonBroadway** [comment]
poo currently sitting at -1 total

**Comment by Arzion5** [comment]
+1, great south american simulator

**Comment by ghost** [comment]
-1
no
toilet humor gone too far [gone wild] [gone bananas]

**Comment by j-mie** [comment]
ok basically all of the headmins are against it, so rippie

**Comment by TungstenOxide** [comment]
Wait what? I say +1

**Comment by bardofrage** [comment]
-1 again!

**Pull request closed without merging**